sample-bundle/README.md

```diff
@@ -0,0 +1,16 @@
+# Sample Nomad Bundle 
+
+To use this bundle first you need a nomad cluster up and running. To get a quick one up and running locally, 
+you can run 
+
+```bash
+sudo nomad agent -dev \
+  -bind 0.0.0.0 \
+  -network-interface='{{ GetDefaultInterfaces | attr "name" }}'
+```
+
+Make note of the IP address of the nomad server, it should get output in the terminal after running the command. 
+Edit the NOMAD_ADDR in the porter.yaml file to match your cluster ip address. Don't forget the protocol (http) and port number.
+
+Next run `porter install` in the directory containing the porter.yaml. This will install the nomad jobs into your cluster.
```

Comment:
I feel like there should be a `porter.yaml` in here as a sample

Reply:
there is